addons/mail/static/src/components/message/message.js

```diff
@@ -448,14 +449,17 @@ class Message extends Component {
             }
             return;
         }
-        this.state.isClicked = !this.state.isClicked;
+        if (!isEventHandled(ev, 'Message.ClickHandled')) {
+            this.state.isClicked = !this.state.isClicked;
+        }
     }
 
     /**
      * @private
      * @param {MouseEvent} ev
      */
     _onClickAuthorAvatar(ev) {
+        markEventHandled(ev, 'Message.ClickHandled');
```

Comment:
The "mark name" should be the name related to the context on which it was handled (here "author avatar") not the "target" of the "mark".

So we should have various names for the various click handlers.

Reply:
This make the test a bit heavy since `isEventHandled`doesn't accept an array of markName.
But I get your point.

Reply:
It's more verbose but it gives way more control to top level components.